### ISSUE TITLE
[Web][Accessibility][11yMAS][Keyboard Navigation][MAS2.1.1][HighImpact][User cannot copy the url to import the snippet using keyboard.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#716]
**User Experience:**
User with vision disability and keyboard only user will not be able to copy the url for importing the snippet using keyboard.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0

**Attachments:**
![mas2 1 1 copy_url](https://user-images.githubusercontent.com/33054846/32716774-416995ea-c87d-11e7-8869-0b5771c1e526.png)

**Repro Steps:**
1. Open the URL: https://script-lab.azureedge.net/#/edit/excel
2. Navigate “New snippet” and select it.
3. Sign in to Github with valid credentials.
4. Navigate to Share ribbon menu.
5. Navigate to “New scret gist”/“New Public gist” and select it.
6. As soon as the “New secret gist”/“New Public gist” is select a ’Share your Snippet’ dialog is opened with the url which is in the form of the text in the dialog.
7. Verify that user cannot copy that url to import the snippet.

**Current Behavior:**
As it is in text format user cannot copy the url for git hub gist using keyboard so that it can be pasted in the text field for the Import. 

**Expected Behavior:**
As a suggestion there should be a button available to copy the url from the dialog from the ‘Share your snippet’ dialog or the URL present should be in the form of the link and not the text.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=2.1.1
